dashboard/views/_positions/_ec/_secretary/_position_management.py

```diff
@@ -1,13 +1,25 @@
 from django.http import HttpResponseRedirect
 from django.shortcuts import render
 from django.urls import reverse, reverse_lazy
-from django.views.generic.edit import UpdateView, DeleteView
 
 from dashboard.forms import PositionForm
 from dashboard.models import Position
-from dashboard.utils import all_positions, verify_position
+from dashboard.utils import verify_position
 
-@verify_position(['Secretary', 'Vice President', 'President', 'Adviser'])
+from dashboard.views._dashboard_generic_views import DashboardUpdateView, DashboardDeleteView
+
+
+# EC Positions
+all_positions = [
+    'president', 'vice-president', 'vphs',
+    'secretary', 'treasurer', 'marshal', 'recruitment-chair',
+    'scholarship-chair', 'service-chair', 'philanthropy-chair',
+    'detail-manager', 'alumni-relations-chair',
+    'memdev-chair', 'social-chair', 'pr-chair'
+]
```

Comment:
ok so

1. the comment about this being EC positions is inaccurate
2. shouldnt we be using `Position.PositionChoices` instead of this?

Reply:
Yeah that comment I think was just left over. I've deleted this entire line and just replaced the single instance in which we used this with "Position.PositionChoices.values" which outputs the same thing